apps/condo/domains/ticket/components/TicketClassifierSelect.tsx

```diff
@@ -1,13 +1,16 @@
 import React, { useEffect, useState, useRef, useMemo } from 'react'
-import { Select, Input, Col, Form, Row } from 'antd'
+import { Select, Input, Col, Form, Row, notification } from 'antd'
 const { Option } = Select
 import { useApolloClient } from '@core/next/apollo'
 import { useIntl } from '@core/next/intl'
 import { uniqBy, isEmpty, find, pick, get } from 'lodash'
-import { ClassifiersQueryLocal, ClassifiersQueryRemote, TicketClassifierTypes } from '@condo/domains/ticket/utils/clientSchema/classifierSearch'
+import { ClassifiersQueryLocal, TicketClassifierTypes } from '@condo/domains/ticket/utils/clientSchema/classifierSearch'
 import { useTicketValidations } from '@condo/domains/ticket/components/BaseTicketForm/useTicketValidations'
 import { Gutter } from 'antd/es/grid/row'
 import { TicketFormItem } from './BaseTicketForm'
+import { useLazyQuery } from '@apollo/client'
```

Comment:
It’s better to import it from core to avoid different Apollo version collision

Reply:
Removed